docs/user-guide/groupby.ipynb

```diff
@@ -195,6 +193,47 @@
     "grouped = sc.groupby(data, group=param, bins=bins).mean('x') # note the lack of quotes around param!\n",
     "sc.plot(grouped)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Usage examples\n",
+    "\n",
+    "### Filtering a variable using `groupby.copy`\n",
+    "\n",
+    "Apart from reduction operations discussed above, `groupby` also supports `copy`, which allows us to extract a group without changes.\n",
+    "We can use this, e.g., to filter data.\n",
+    "This can be used for filtering variables:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var = sc.array(dims=['x'], values=np.random.rand(100))\n",
+    "select = var < 0.5 * sc.Unit('')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We use a temporary data array pass `1` to `copy` to extract the second group (group indices start at 0).\n",
```

Comment:
Sentence doesn't fully make sense.
```suggestion
    "We pass `1` to `copy` to extract the second group (group indices start at 0).\n",
```

Reply:
"temporary data array" refers to `sc.DataArray(var, coords={'dummy':select})`, are you sure it should be removed?

Reply:
Written more verbosely and clearly now.